lib/assessment/rubric_tester.py

```diff
@@ -180,6 +177,11 @@ def get_s3_folder(s3, path_from_s3_root):
             continue
         bucket.download_file(obj.key, target)
 
+# Clone params from private github repository. Requires ssh key.
+def get_params_github():
+    git_repo = "git@github.com:code-dot-org/aitt_release_data.git"
+    subprocess.run(["git", "clone", git_repo], capture_output=True, cwd='~')
```

Comment:
I needed this change to make it work locally, copied from wherever else you accessed `~`:
```suggestion
    subprocess.run(["git", "clone", git_repo], capture_output=True, cwd='~')
    subprocess.run(["git", "clone", git_repo], capture_output=True, cwd=os.path.expanduser('~'))
```